sdk/process/keeper.go

```diff
@@ -6,21 +6,21 @@ import (
 	cosmostypes "github.com/cosmos/cosmos-sdk/types"
 	"github.com/mesg-foundation/engine/codec"
 	"github.com/mesg-foundation/engine/hash"
-	"github.com/mesg-foundation/engine/ownership"
+	ownershippb "github.com/mesg-foundation/engine/ownership"
 	"github.com/mesg-foundation/engine/process"
 	instancesdk "github.com/mesg-foundation/engine/sdk/instance"
-	ownershipsdk "github.com/mesg-foundation/engine/sdk/ownership"
+	"github.com/mesg-foundation/engine/x/ownership"
 )
 
 // Keeper is the service keeper.
 type Keeper struct {
 	storeKey        *cosmostypes.KVStoreKey
-	ownershipKeeper *ownershipsdk.Keeper
+	ownershipKeeper ownership.Keeper
```

Comment:
If we really want to do it nicely, the process module should declare an `OwershipKeeper` interface. But i'm ok to keep it like this for now.

Reply:
Yes, that's the standard way for the framework and it will be done when process/service module will be transform to it.